lib/commands/outdated.js

```diff
@@ -1,5 +1,6 @@
-const os = require('os')
-const { resolve } = require('path')
+const os = require('node:os')
```

Comment:
Is there a plan to update all node modules require statements with the prefix?

Reply:
So far no, just updating as we go.